README.md

```diff
@@ -0,0 +1,89 @@
+Django-locking
+==============
+
+About
+-----
+
+`django-locking` provides a system that makes concurrent editing impossible, and informs users of what other users are working on and for how long that content will remain locked. Users can still read locked content, but cannot modify or save it.
+
+Requirements
+-------------
+
+* Django 2.2
+* Python 3.6 - 3.8
+
+Installation
+------------
+
+1. Download and install django-locking.
+
+    ```console
+    $ pip install git+git://github.com/unt-libraries/django-locking.git
+    ```
+
+2. Add app to INSTALLED_APPS.
+
+    ```python
+    INSTALLED_APPS = (
+        'locking',
+    )
+    ```
+
+Developing
+----------
+
+1. Clone the git repository:
+
+    ```python
+        git clone https://github.com/unt-libraries/django-locking.git
+    ```
+
+2. Navigate into the cloned repository:
+
+    ```python
+        cd django-locking
+    ```
+
+3. Install the requirements (preferably in a virtual environment):
+
+    ```python
+        python setup.py install
+    ```
+
+4. Run the migrations:
+
+    ```python
+        django-admin migrate
```

Comment:
I installed Django 2.2 (should that be something added to `install_requires`?) then when I run this I get:
```
django.core.exceptions.ImproperlyConfigured: Requested setting DATABASES, but settings are not configured. You must either define the environment variable DJANGO_SETTINGS_MODULE or call settings.configure() before accessing settings.
```
Is there an expectation certain things are already done to have these commands work?

Reply:
oh...yeah I had to set environment variables in my virtualenv before running these commands using - 

`export DJANGO_SETTINGS_MODULE=locking.settings`
`export PYTHONPATH="$(pwd)"`

Reply:
Do either of you think that we should have a `manage.py` and use manage.py commands to run and test the app similar to other apps?

Reply:
That did cross my mind--I think it would be ok to add this.